packages/world-modules/src/modules/erc20-puppet/ERC20Module.sol

```diff
@@ -32,16 +31,14 @@ contract ERC20Module is Module {
 
   function _requireDependencies() internal view {
     // Require PuppetModule to be installed
-    if (InstalledModules.get(PUPPET_MODULE_NAME, keccak256(new bytes(0))) == address(0)) {
+    if (!isInstalled(PUPPET_MODULE_NAME, abi.encode(keccak256(new bytes(0))))) {
```

Comment:
I think we're doing `keccak256` twice now (once here, once in `isInstalled`). Should this instance be removed?
```suggestion
    if (!isInstalled(PUPPET_MODULE_NAME, abi.encode(new bytes(0)))) {
```